lib/bulk/common.js

```diff
@@ -1,8 +1,7 @@
 'use strict';
 
-const {
-  BSON: { Long, ObjectId }
-} = require('../deps');
+const { BSON } = require('../deps');
+const { ObjectId, Long } = BSON;
```

Comment:
So this I broke out for an opinionated reason and totally can revert, but @mbroadst is working on fixing the bug that is blocking us from generating type definitions from all this jsdoc however it may not be fixable in the near future.

Since an alternative would be to change up the way we import and export things I investigated that es6 module `import`/`export` syntax particularly `import { BSON } from '../deps';` while it looks like object de-structuring it isn't related at all apparently so it does not support `import { BSON: {ObjectId, Long} } from '../deps';`, this is me jumping the gun and cleaning up that behavior in advance.
Also I don't love that it breaks it across two lines.
Like I said TOTALLY opinionated but with some reasoning behind it 😅